DEVELOPMENT.md

```diff
@@ -0,0 +1,9 @@
+# Developing the Git Town action source code
+
+This page helps you get started hacking on the Git Town codebase.
+
+## setup
+
+1. install Node.js v18+
```

Comment:
In `package.json` it runs esbuild with `--target=node20`. Would it make sense to update the documented Node version to 20 here?

Reply:
Yep definitely, good catch! I'll include it in [package.json.engines](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines) as well to help enforce that consistency.